plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -382,10 +393,25 @@ private PipelineOptionsHierarchy getPipelineOptionsHierarchy(
     return pipelineOptionsHierarchyFactory.global(monitor);
   }
 
+  private IProject getProject(ILaunchConfiguration configuration) {
+    try {
+      String projectValue = configuration.getAttribute(
+          IJavaLaunchConfigurationConstants.ATTR_PROJECT_NAME, "");
+      return getProject(projectValue);
+    } catch (CoreException ex) {
+      // BTW, I don't think this can really happen.
```

Comment:
Yeah, maybe this can happen when having trouble reading a value from a launch config saved on disk.